spec/rubocop/target_finder_spec.rb

```diff
@@ -440,6 +440,21 @@
       expect(found_basenames).to include('ruby3.rb')
       expect(found_basenames).to include('ruby4.rb')
     end
+
+    # Cannot create a directory with containing `*` charactor on Windows.
```

Comment:
It looks like a typo :-)

```suggestion
    # Cannot create a directory with containing `*` character on Windows.
```

Reply:
Oops...Thank you.